FirebaseABTesting/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+#v3.2.1
+- [added] Completion handler for FIRExperimentController's updateExperimentsWithServiceOrigin method.
+- [deprecated] FIRExperimentController updateExperimentsWithServiceOrigin:events:policy:lastStartTime:payloads:.
```

Comment:
Repeat the verb even if it's in the tag for the sake of complete sentences. `Added completion handler ...` and `Deprecated FIRExperimentController ...`. Also, prefer Swift method syntax over objc: `FIRExperimentController.updateExperiments(serviceOrigin:events:policy:lastStartTime:payloads:)`